recipes/calcephpy/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "calcephpy" %}
+{% set version = "3.0.0" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "e8b2e08306412355672ee945173bad3b67ce607558213945e061a86ce2af3107" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
```

Comment:
Using Jinja for parameter names is not common, and IMHO hurts readability

Reply:
In the same way, the command "conda skeleton pypi calceph" generates these custom command inside the meta.yaml. I prefer to keep that generated command if you are ok.